torch/_inductor/standalone_compile.py

```diff
@@ -58,7 +58,7 @@ def __init__(
         self._artifacts = artifacts
 
     def __call__(self, *args: Any) -> Any:
```

Comment:
If you use a ParamSpec and a return TypeVar here when you capture the function, you might be able to preserve arg typing and return type here.